src/Umbraco.Core/Services/Implement/MediaService.cs

```diff
@@ -535,6 +535,30 @@ public IEnumerable<IMedia> GetAncestors(IMedia media)
                 return GetPagedLocked(GetPagedDescendantQuery(null), pageIndex, pageSize, out totalChildren, filter, ordering);
             }
         }
+        /// <inheritdoc />
+        public IEnumerable<IMedia> GetPagedDescendants(int id, long pageIndex, int pageSize, 
```

Comment:
This code can be shared in one method which can accept a bool as to whether or not to include the count